requirements.txt

```diff
@@ -1,4 +1,4 @@
-Django==1.4.18
+Django==1.7.7
```

Comment:
since we don't have tests, there isn't much to do besides upgrade and try everything out.  Then fix any 500s that come up.